packages/util/src/string.ts

```diff
@@ -109,3 +109,22 @@ export function utilSafeString(str) {
 export function utilUniqueString(val) {
   return 'ideditor-' + utilSafeString(val.toString()) + '-' + new Date().getTime().toString();
 }
+
+// Returns a comparator function for sorting strings alphabetically in ascending order,
+// regardless of case or diacritics.
+// If supported, will use the browser's language sensitive string comparison, see:
+// https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Intl/Collator/Collator
+export function utilSortString(locale = 'en'): (x: string, y: string) => number {
+  if (typeof Intl === 'object' && 'Collator' in Intl) {
+      return (new Intl.Collator(locale, {
+          sensitivity: 'base',
+          numeric: true
```

Comment:
Should these options be configurable? I could see this function being useful for sorting localized field options with different sensitivity.

Reply:
It probably should be configurable, but I think we can accept it as-is and iterate on it if needed..